packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getViewer/getViewer.spec.js

```diff
@@ -7,36 +7,44 @@ import SingleImageViewer from '../../components/SingleImageViewer'
 import SingleTextViewer from '../../components/SingleTextViewer'
 import SubjectGroupViewer from '../../components/SubjectGroupViewer'
 
```

Comment:
Are you going to add a test case for the volumetric viewer to this file?

Reply:
Yes - I did this in the [SubjectViewer.spec.js](https://github.com/zooniverse/front-end-monorepo/pull/6384/commits/5b9a052bb56e4d7dc1270d42b1b77aa083c5210e#diff-10013e86b38c949dc41526e31b6e5849fa2cf78f605d9471f04e57833a6b0892R55-R70) follow up PR #6384. The reason being is I wanted this PR to get the build process correct (i.e. import process, build process, test fixes for async shift) without focusing on making sure it could render/display. That's what the follow up PR does.